ResourceStatusSystem/Utilities/InfoGetter.py

```diff
@@ -270,32 +115,29 @@ def __getPoliciesThatApply( self, decisionParams ):
 
       # We are not going to use name / type anymore, but we keep them for debugging
       # and future usage.
-      policyDict = {
-                     'name' : policyName,
+      policyDict = { 'name' : policyName,
                      'type' : policyType,
-                     'args' : {}
-                   }
+                     'args' : {} }
```

Comment:
If we agree to follow pep8 as I think we did, the previous convention is the correct one